google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -806,7 +806,7 @@ public Timestamp writeAtLeastOnce(Iterable<Mutation> mutations) throws SpannerEx
                 new Callable<CommitResponse>() {
                   @Override
                   public CommitResponse call() throws Exception {
-                    return rawGrpcRpc.commit(request, options);
+                    return gapicRpc.commit(request, options);
```

Comment:
does gapicRpc do retries? We do not do any retries in GrpcSpannerRPC and instead they are done at a higher level. Is that behavior preserved here.

Reply:
gapicRpc does retry. I take this to mean we might retry more than we want?

Reply:
So ideally we would want to get rid of the higher level retry. But last time we discussed this, I think not all the things we retry on were supported in GAPIC. The problem with retrying in two layers is that exponential backoff gets messed up. Also in spanner code we keep on retrying till the deadline specified in the grpc `Context` has elapsed. We would like to preserve that behavior.

Reply:
@hzyi-google I think the best thing to do at this point is to use `SpannerStubSettings.applyToAllUnaryMethods` and make all methods not retry. Then we can use the existing retry. Then we can improve gax and get rid of the higher level one after.

Reply:
Yeah I agree.